app/src/main/kotlin/com/wire/android/ui/home/newconversation/NewConversationViewModel.kt

```diff
@@ -162,23 +168,73 @@ class NewConversationViewModel
         }
     }
 
+    fun onGroupNameChange(newText: TextFieldValue) {
+        when {
+            newText.text.trim().isEmpty() -> {
+                groupNameState = groupNameState.copy(
+                    animatedGroupNameError = true,
```

Comment:
Does it mean that it will be animated at the start? Maybe it would be better to change the condition to something like groupNameState.groupName.text.isNotEmpty() && newText.text.trim().isEmpty() so it will animate only when someone cleared the previously entered text. I'm not sure if the shake animation is needed in such case, maybe just disabling the "continue" button is enough? 🤔

Reply:
no it won't have the error or the animation at the begging , it will only appear when the user type something and delete again , hmmm about only disabling the button that also work , but I saw this animation used somewhere else , so I kept it to make things consistent